gammasim/simtools
# Check suffix of files in schemas/model_parameters

All files in 'simtools/schemas/modelparameters' should have the file suffix `*.schema.yml`. Add here a test that this is actually the case.

Background is the correction required in #1179, which lead to some non-obvious issues when using these schema files in the model_parameter validation.

To test that this works: 

- see the failing CI-schema-validation workflow in https://github.com/gammasim/simtools/actions/runs/11126742313/job/30917377243 (executed before merging #1179)
- see the successful CI-schema-validation workflow in https://github.com/gammasim/simtools/actions/runs/11126797966/job/30917570189 (executed after merging #1179)
